plugins/src/desktop_entries/utils.rs

```diff
@@ -36,3 +36,13 @@ pub fn get_description<'a>(de: &'a DesktopEntry<'a>, locales: &[String]) -> Stri
         None => desc_source,
     }
 }
+
+// todo: cache
+#[must_use]
+pub fn is_session_cosmic() -> bool {
+    if let Ok(var) = std::env::var("XDG_CURRENT_DESKTOP") {
+        return var.contains("COSMIC");
+    }
+
+    false
+}
```

Comment:
I feel like in the future it would be nicer if the plugin.ron files would have an "AlwaysShowIn"-key similar to desktop files, so that other desktops could more easily extend this, if they support the necessary protocols. (Why do the plugins not use desktop-files in the first place?)

Anyway, this is good enough for now.